### PR TITLE
#160597069 Add guest room and bed during request 

### PR DESCRIPTION
**Pull request by ezrqnkemboi**
#### What does this PR do?

This PR ensure that user enters available room and bed space during new travel request creation.

#### Description of Task to be completed?

As a user, I should be able to add the room and bed space during my new travel request. This will ensure that if I will not get any room available, they will look for alternatives or postpone trip duration. 
It makes sure that room selected is based on user destination, gender and travel duration. 

#### How should this be manually tested?

- Clone the repo - git clone https://github.com/andela/travel_tool_front.git
- Change into the project directory
- git checkout ft-check-available-room-bed-160597069
- Run the server on [this PR](https://github.com/andela/travel_tool_back/pull/92)
- Login and make a new request
- Make three trips, ( oneWay, Multi and return ) trips
- After filling the details on making a new request, when checking for available rooms, the room name and bed name available are shown. 

#### Any background context you want to provide?
Ensure that you have guest houses that has rooms and beds. 

#### What are the relevant pivotal tracker stories?

[#160597069](https://www.pivotaltracker.com/story/show/160597069)

#### Screenshots (if appropriate)

screen shot 2018-10-18 at 17 03 27

### CODE REVIEW

File: src/components/Forms/NewRequestForm/FormFieldsets/TravelDetails.jsx [hunk 1]
```diff
@@ -60,64 +57,123 @@ class TravelDetailsFieldset extends Component {
...
+      parentIds,
+      existingTrips,
+      fetchAvailableRooms,
+      availableRooms
+    } = this.props;
     const { renderInput } = this.inputRenderer;
+    let data = {};
+    let bedChoices = availableRooms.beds || ['Fetching beds...'];
+    bedChoices = bedChoices.map(choice => ({
+      label: `${choice.rooms.roomName}, ${choice.bedName}`,
```
[the remark this answers is not in the document]
Comment (ezrqnkemboi) [reply src/components/Forms/NewRequestForm/FormFieldsets/TravelDetails.jsx]: Checking on that and rectifying.

**Comment by ezrqnkemboi** [comment]
@johnmutuma5 I have realized that I will try to fix them.

File: src/components/Forms/NewRequestForm/FormFieldsets/TravelDetails.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (ezrqnkemboi) [reply src/components/Forms/NewRequestForm/FormFieldsets/TravelDetails.jsx]: I have tried to rectifying in that manner, but, It includes the guest house name, that is not needed. 
screen shot 2018-10-18 at 18 32 03

The Mock up: 
screen shot 2018-10-18 at 18 33 49

[the remark this answers is not in the document]
Comment (samuelkubai) [reply src/components/Forms/NewRequestForm/FormFieldsets/TravelDetails.jsx]: I think we can go with the current implementation for now and circle back for a refactor.

[the remark this answers is not in the document]
Comment (ezrqnkemboi) [reply src/components/Forms/NewRequestForm/FormFieldsets/TravelDetails.jsx]: That is better.

**Pull request merged**